Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Applications/WindowsApp/AqualityServices.cs

```diff
@@ -31,9 +31,9 @@ private static WindowsApplication StartApplication(IServiceProvider services)
                 var driverService = AppiumLocalService.BuildDefaultService();
                 driverService.Start();
                 driverServer = driverService.ServiceUrl;
-                options.AddAdditionalCapability("platformVersion", "10");
-                options.AddAdditionalCapability("platformName", "Windows");
-                options.AddAdditionalCapability("deviceName", "WindowsPC");
+                options.PlatformVersion = "10";
+                options.PlatformName = "Windows";
+                options.DeviceName = "WindowsPC";
```

Comment:
it seems that you need to set AutomationName property since you use WAD with Appium instead of using it directly